src/sysext/sysext.c

```diff
@@ -474,11 +518,38 @@ static int verb_status(int argc, char **argv, void *userdata) {
         return ret;
 }
 
+static int append_overlayfs_path_option(
+                char** options,
+                const char* separator,
+                const char* option,
+                const char* path) {
+
+        _cleanup_free_ char *escaped = NULL;
+
+        assert (options);
+        assert (separator);
+        assert (path);
+
+        escaped = shell_escape(path, ",:");
+        if (!escaped)
+                return log_oom();
+
+        if (option) {
+                if (!strextend(options, separator, option, "=", escaped))
+                        return log_oom();
+        } else if (!strextend(options, separator, escaped))
```

Comment:
use strextend_with_separator()?

Reply:
Doesn't seem to be useful here though. `strextend_with_separator(options, separator, "", escaped)`?